apps/docs/content/components/navigation/Disclosure.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Disclosure
+description: The disclosure component is used to put long sections of information under a block that users can expand or collapse.
+category: "navigation"
+links:
+    source: https://github.com/gsoft-inc/wl-hopper/blob/main/packages/components/src/Disclosure/src/Disclosure.tsx
+    aria: https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/
+---
+
+<Example src="Disclosure/docs/preview" isOpen />
+
+<FeatureFlag flag="next">
+    ## Guidelines
+
+    TODO: If we have some guidelines about this component&apos;s usage
+
+    ### Accessibility ?
+
+    TODO: If we have some guidelines about this component and accessibility
+</FeatureFlag>
+
+## Anatomy
+
+<FeatureFlag flag="rc">
+    TODO: We have anatomy screenshots from the Figma, we could most likely use them here
+
+    ### Concepts
+
+    TODO: links to related concepts
+</FeatureFlag>
+
+### Composed Components
+
+A `Link` uses the following components.
+
+<ComposedComponents components={["Text"]}/>
+
+## Usage
+
+### Disabled
+
+A disclosure can be disabled.
+
+<Example src="Disclosure/docs/disabled" />
+
+### Inline
+
+To make a disclosure inline, set the `variant` prop to `inline`. 
+This is typically used when placing a disclosure inside a container. In this example, it's inside a `Card` component.
+
+<Example src="Disclosure/docs/inline" />
+
+### Icon
+
+Disclosures can contain an icon.
+
+<Example src="Disclosure/docs/icon" />
+
+### Description
+
+Disclosures can contain a description.
+
+<Example src="Disclosure/docs/description" />
+
+### Controlled
+
+A disclosure can handle its expanded state in controlled mode.
+
+<Example src="Disclosure/docs/controlled" />
+
+### Custom Header
```

Comment:
we shoud mention that using Button from RAC is fine also. It's not only for Hopper components. Using RAC button allows to start from scratch, instead of using premade style for the containing header